cucumber-expressions/java/pom.xml

```diff
@@ -17,29 +17,32 @@
 
     <properties>
         <project.Automatic-Module-Name>io.cucumber.cucumberexpressions</project.Automatic-Module-Name>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
     </properties>
 
     <scm>
         <connection>scm:git:git://github.com/cucumber/cucumber-expressions-java.git</connection>
         <developerConnection>scm:git:git@github.com:cucumber/cucumber-expressions-java.git</developerConnection>
         <url>git://github.com/cucumber/cucumber-expressions-java.git</url>
-        <tag>HEAD</tag>
     </scm>
 
+    <build>
```

Comment:
This is already defined in https://github.com/cucumber/cucumber-parent/blob/85620180cb6d6a802dd42bb0c5bb19ad6793bab4/pom.xml#L179-L188

Can we remove this?

Reply:
Yeah. Should come from the latest parent pom.